<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inception Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/light105.png</normaloff>:/icons/icons/light105.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mainarea" native="true">
      <layout class="QHBoxLayout" name="mainAreaTopLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>529</width>
            <height>558</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="mainAreaLayout">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExport_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionCursor"/>
   <addaction name="actionInsert"/>
  </widget>
  <widget class="QDockWidget" name="optionsDockWidget">
   <property name="floating">
    <bool>true</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="imageFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="fileBrowseButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="insertImageLabel">
          <property name="text">
           <string>Choose an image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>-1</number>
        </property>
        <property name="verticalSpacing">
         <number>-1</number>
        </property>
        <item row="4" column="0">
         <widget class="QLabel" name="mattingLabel">
          <property name="text">
           <string>Matting method</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="mattingComboBox">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>floodfill</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>simplematting</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>closedform</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>grabcut</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="mergeLabel">
          <property name="text">
           <string>Merge method</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="mergeComboBox">
          <item>
           <property name="text">
            <string>merge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>poisson</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="colorAdjustCheckBox">
          <property name="text">
           <string>Color adjust</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="shadowsCheckBox">
          <property name="text">
           <string>Shadows</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/selection7.png</normaloff>:/icons/icons/selection7.png</iconset>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/cursor12.png</normaloff>:/icons/icons/cursor12.png</iconset>
   </property>
   <property name="text">
    <string>cursor</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/light105.png</normaloff>:/icons/icons/light105.png</iconset>
   </property>
   <property name="text">
    <string>insert</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/left3.png</normaloff>:/icons/icons/left3.png</iconset>
   </property>
   <property name="text">
    <string>move</string>
   </property>
  </action>
  <action name="actionPaint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/paint87.png</normaloff>:/icons/icons/paint87.png</iconset>
   </property>
   <property name="text">
    <string>paint</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/stamp11.png</normaloff>:/icons/icons/stamp11.png</iconset>
   </property>
   <property name="text">
    <string>clone</string>
   </property>
  </action>
  <action name="actionExport_As">
   <property name="text">
    <string>Export As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
